src/components/card/Card.tsx

```diff
@@ -12,34 +13,68 @@ interface CardProp {
 }
 
 function Card({ img, name, slug, tea, description, addFavs, favs }: CardProp) {
+
+  const [targetTea, setTargetTea] = useState<any>({})
+  const [isClicked, setIsClicked] = useState<boolean>(false)
   const [color, setColor] = useState(
     favs.some((fav) => fav.slug === slug) ? "#895B1E" : "#B1AE91"
   );
 
+  const css = {
+    transform: 'rotateY(180deg)',
+  }
+
+  const ncss = {
+    transform: 'rotateY(0deg)'
+  }
+
+
   function favTea(e: React.MouseEvent<HTMLButtonElement>) {
     addFavs(tea);
     setColor(color === "#B1AE91" ? "#895B1E" : "#B1AE91");
   }
 
+  async function fetchTea(e: React.MouseEvent<HTMLDivElement>): Promise<any> {
+    let tea = await fetchSingleTea(slug)
+    let target = tea[0]
```

Comment:
Still chuckling at the array of one solitary item . Thanks for lending your creativity!